src/modules/launcher/Wox.Infrastructure/StringMatcher.cs

```diff
@@ -223,6 +223,12 @@ private static int CalculateSearchScore(string query, string stringToCompare, in
                 }
             }
 
+            // To increase the score of an exact match
```

Comment:
nit:   It would be better to use a variable or constant here to have self documenting code.  eg. 
```
var bonusForExactMatch = 10; 
score += bonusForExactMatch; 
```